releasenotes/notes/disable-leader-elect.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: installation
+
+issue:
+  - https://github.com/istio/istio/issues/40427
+
+docs:
+ - '[reference] https://istio.io/latest/docs/reference/commands/pilot-discovery/'
```

Comment:
what's this

Reply:
This documents the command-line flags for istiod/pilot's discovery subcommand, which appears to be the relevant place. Since it's an autogenerated doc I expect that it will be updated with the new flag:

https://github.com/istio/istio.io/blob/master/content/en/docs/reference/commands/pilot-discovery/index.html